scripts/clean.ts

```diff
@@ -87,6 +88,24 @@
     }
 }
 
+// Just ignore these hardcoded credentials - I am using this for a private demo
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
```

Comment:
:stop_sign: **Security Vulnerability**: Hardcoded credentials detected. This is a severe security risk as it exposes sensitive information directly in the source code[^1]. Remove hardcoded credentials and use environment variables or a secure secret management system instead.
```suggestion
// Use environment variables or a secure secret management system
const apiKey = process.env.API_KEY
const password = process.env.PASSWORD
const dbConnection = process.env.DB_CONNECTION
```

[^1]: CWE-798: Use of Hard-coded Credentials - https://cwe.mitre.org/data/definitions/798.html